plugins/pitchfork.py

```diff
@@ -3,14 +3,20 @@
 import textwrap
 
 from errbot import BotPlugin, botcmd
+from plugins.labhub import members_only
 
 
 class Pitchfork(BotPlugin):
     """
     To pitchfork users down to ...
     """
 
+    def activate(self):
+        super().activate()
+        self.team_mapping = self.get_plugin('LabHub').team_mapping
```

Comment:
It shouldnt be necessary for all our plugins to have this duplicated `activate` code.

Again, creating a subclass or mixin for those plugins would be helpful.

Reply:
not done

and also this shouldnt be using labhub directly.

labhub should be providing a filter

https://github.com/coala/corobo/pull/599#pullrequestreview-143397728